gfx/MShockXotto+/pngs_tiles_32x32/mods/mindovermatter/monsters/feral_lab_psychics/mon_feral_scentist_psychic.json

```
@@ -0,0 +1,10 @@
+[
+  {
+    "id": [ "mon_feral_scientist_psychic" ],
+    "fg": [
+      { "weight": 1, "sprite": "mon_feral_scientist_psychic" }
+    ],
```

```suggestion
    "fg": "mon_feral_scientist_psychic",
```
You can just have it like this, no need for the complicated part